R/generateEDGEdata.R

```diff
@@ -5,26 +5,27 @@
 #' @param output_folder folder hosting REMIND input files
 #' @param EDGE_scenario EDGE transport scenario specifier
 #' @param REMIND_scenario SSP scenario
+#' @param IEAbal use mrremind generated data: in case of a REMIND preprocessing run, load population
```

Comment:
here it would be nice to explicitly mention the call to mrremind, i.e., `mrremind::calcOutput("Population")`

Reply:
should be solved by next commit.